### DISCUSSION TITLE
Issue with testing the calculator

### DISCUSSION THREAD

**Discussion by JohnLSison** [srothetafesa/Tafe24S1T109_Ext#6] [General]
Hey Sam, 
could you please check my xaml and c# codes, i was trying to test it yesterday but my visual studio was having a fit and i couldn't actually see if the menu was working or not.
My xamarim.forms was not working at all and i had to do the coding through notepad++ for the xaml.

Cheers,
John

**Comment by srothetafesa** [comment]
Hi John,

I just checked and it works fine on my PC.

Cheers,
Sam
